utils/s2n_rfc5952.h

```diff
@@ -0,0 +1,23 @@
+#pragma once    
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ */
+
+struct s2n_blob;
```

Comment:
we should just include the right header :)

Reply:
Oh, I think that was PTSD from my C++ days, where I have to wait years for a build so I got in the habbit of forward declaring everything. Can move it over.